apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -267,15 +267,15 @@ function wpcom_display_global_styles_launch_bar( $bar_controls ) {
 		$preview_text     = __( 'Hide premium styles', 'full-site-editing' );
```

Comment:
Should we also change this for consistency?
```suggestion
		$preview_text     = __( 'Hide custom styles', 'full-site-editing' );
```